runtime/doc/vim_diff.txt

```diff
@@ -277,7 +279,16 @@ Variables:
   |v:windowid| is always available (for use by external UIs)
 
 ==============================================================================
-4. Changed features					 *nvim-features-changed*
+4. Upstreamed features					 *nvim-upstreamed*
+
+These Nvim features were later integrated into Vim.
+
+- |:source| works anonymous (no file) scripts
```

Comment:
Instead of removing items from vim_diff.txt, move them here.

Reply:
Also add things like `<Cmd>`, `WinClosed`, `WinScrolled`, and some 'fillchars' flags here?